### ISSUE TITLE
New version 2.1.4 - 32 bit - execution error -  cannot proceed because of MSVCR120D.dll

### ISSUE THREAD

**Issue by Irockus** [elieserdejesus/JamTaba#1032]
Running windows 10 pro 64 bit OS  (version 1709, OS Build 16299.248)

When I try to run the 32 bit 2.1.4 version, I get the error
cannot proceed because of MSVCR120D.dll

Compiled with debug library maybe ?   

tried uninstalling completely and re-installing - same error
tried reinstalling some visual  studio 2012 and 2013 x86 redistributables, but they did not include the "D" version of the .dll

FYI - I got the 64 bit version of 2.1.4 to work, but only after uninstalling the previous 2.1.3 build completely first.

**Comment by Irockus** [comment]
![image](https://user-images.githubusercontent.com/32970293/37254858-c62036fe-2511-11e8-841d-ce9af7f4114d.png)

**Comment by elieserdejesus** [comment]
Hi @Irockus , thanks for your error report.

I tried reproduce the problem here (Win 10 64 bits). 

1 - I uninstalled my JTBA v2.1.4 64 bits
2 - Installed the v2.1.3 32 bits, started without problem
3 - Installed the v.2.1.4 32 bits, started without problem after installed.

@jonjamcam , I can't reproduce the problem reported by @Irockus . Can you try reproduce in your machine?

**Comment by Irockus** [comment]
I decided to start from scratch ...

I uninstalled all previous jamtaba versions, both 32 and 64 bit.  clean system

1.  Started with V 2.1.4 , 32 bit,  got the error when I tried to run it - msvcr120d.dll not found.  I cannot find this file on my system and I believe it is a debug version of the visual studio runtime lib.

2.  I installed V 2.1.4 64 bit, got it working no problem.

3.  I installed V 2.1.3 - 32 bit, got it working no problem

4.  re-installed V 2.1.4 - 32 bit, got same error

Went to this site
https://www.dll-files.com/msvcr120d.dll.html

downloaded this version
MSVCR120D.DLL, FILE DESCRIPTION: MICROSOFT® C RUNTIME LIBRARY
32 bit - 12.0.20827.3 | 1.74 MB | 0.67 MB | U.S. English | Microsoft® C Runtime Library

Placed the file in C:\Windows\SysWOW64

All Good !!

V 2.1.4 - 64 bit and 32 bit versions working - sweeeeeet !!!

Some suggestions:

Indicate 32 bit or 64 bit in the "help, current version" - aka About)
Also in the desktop icon name

Ability to download prior version(s) from site, had to look around for my old V 2.1.3  :)

Fantastic Application and great advances indeed !

Run a Private Server feature in Jamtaba ...
I can open another inquiry if you wish, I am curious if one can control the private server new feature - eg server.cfg.  I tried it out and it works but could not establish a UPnP connection through my router - I have the ninjam private server stuff on my machine already and have been able to get that succesfully exposed, etc.

Great work Eliezer and team !

Sincerely
Irockus  aka Rocky








I

**Comment by elieserdejesus** [comment]
@Irockus , thanks for add more details. I will double check the debug DLL issue, this is not expected.

About the private server: the feature is very basic at moment, just the minimum to have some fun with friends. Please open a new issue, just to keep the ideas organized and easy to remember.

> Indicate 32 bit or 64 bit in the "help, current version" - aka About)
Also in the desktop icon name

> Ability to download prior version(s) from site, had to look around for my old V 2.1.3 :)

Good ideas too, please create new issues (one issue per idea). Feel free to create new issues when you have new ideas, you are welcome!
